kubernetes/cray-etcd-operator/Chart.yaml

```diff
@@ -3,4 +3,4 @@ appVersion: 0.12.0
 description: An extension of the official etcd-operator helm chart
 name: cray-etcd-operator
 home: "cloud/cray-charts"
-version: 0.15.1 # We have to rev this anytime we change things
+version: 0.16.2 # We have to rev this anytime we change things
```

Comment:
for a backport to a fix branch we usually don't bump the minor number because that might be used in a newer release.

Reply:
agreed -- refreshing the branch brought the version to 0.16.1 -- where the pod priority change was.  i bumped to 0.16.2 for the backport.  1.2 has moved on to 0.17.* -- so we should be good

Reply:
For 1.0 release, it should be safer to use 0.15.2 instead of 0.16.x to leave a minor version between 1.0 and 1.2 releases.

Reply:
ok -- you've talked me into it, updating to 0.15.2...

Reply:
fixed